tests/Feature/Http/Controllers/Auth/EmailVerificationTest.php

```diff
@@ -93,7 +93,7 @@ public function testEmailIsNotVerifiedWithInvalidHash(): void
     {
         $this->markTestSkipped();
 
-        $user = User::factory()->create([
+        $user = AuthorizedUser::factory()->create([
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create([
````